client/src/pages/TurnsHistory/TurnTable.js

```diff
@@ -10,7 +10,9 @@ import {
   Title,
 } from './styled'
 
-const TurnsTable = ({ turnHistory, turnUser }) => {
+const TurnTable = ({ turnUser, turnHistory }) => {
+  const ascendingTurns = turnHistory.reverse()
```

Comment:
I think you'll have the same problem because you're still mutating the original array. You need to make a copy of it first, then reverse the copy.

```
const turnsCopy = [...turnHistory]
const ascendingTurns = turnsCopy.reverse()

```